docs/aca/09-aca-autoscale-keda/index.md

```diff
@@ -151,16 +154,18 @@ The message structure our backend processor expects is as JSON below, so copy th
 ```
 ![svcbus-send](../../assets/images/09-aca-autoscale-keda/svs-bus-send.jpg)
 
-##### 4. Verify that multiple replicas are created
+##### 4. Verify that Multiple Replicas Are Created
+
+If all is setup correctly, 5 replicas will be created based on the number of messages we generated into the topic. There are various ways to verify this:
```

Comment:
I would actually move this under !!! success to make it explicit and consistent with rest of the success pages.